policy/delegations.md

```diff
@@ -0,0 +1,222 @@
+# 1. Juno Delegation Program
+
+## 1.1 Juno Delegations SubDAO Inception
+
+The Juno Delegation Program (JDP) is part of the Juno Delegations SubDAO, which was established through Juno governance according to [proposal #50](https://www.mintscan.io/juno/proposals/50 "Juno Network Governance Proposal no. 50 on Mintscan").
+
+## 1.2 Why do we have a delegation program?
+
+The three core components of any distributed proof of stake blockchain are delegators, validators and the founding team. As the blockchain matures, additional roles are required to assist with community engagement and support to build on foundations set from genesis. Typical members/roles added include developers, marketing managers, and community managers. As Juno is a DAO-based blockchain, managing delegations to validators from the Juno Development Fund has been (primarily) moved from C-Root to the Juno Delegation SubDAO.
+
+## 1.3 The objectives of the Juno Delegation Program are:
+
+***1.3.1*** To provide the Juno Development Fund with an ongoing income stream from staking rewards which will be used for the future development of the Juno Network.
+
+***1.3.2*** Reward validators that contribute to the Juno Network, via staking JUNO with them, which provides a continual revenue stream via commission.
+
+## 1.4 Who can apply for the delegation program?
+
+Any entity running a validator node on the Juno Network that satisfies the minimum requirements outlined in section 2.1
+
+## 1.5 How can a validator apply for a delegation?
+
+By submitting a delegation request [here](https://forms.gle/jkGKcxud2vswEA7Y8). The form will only be open for submissions during delegation application collection periods. These periods will be announced on the @JunoNetwork Twitter account.
+
+## 1.6 Delegation epoch
+
+Delegations will be made for a set time period. This period is known as an epoch. The epoch will be marked by a Juno Network block number which will be provided after applications are scored and delegations are made. The epoch length is determined by the SubDAO.
```

Comment:
Should we specify what an "epoch" is? I couldn't see a duration specified.

Reply:
We could do this as a vote in the DAO.

IMO, we should be looking at minimum of 3 months initially, and then 6 months once we have more established data from the "previous epoch", at the moment we dont have any data for uptime, etc.

Reply:
Yep, happy to vote on the first duration. I think it should be somewhere in the doc (once we've voted) to inform the validators of the initial delegation period for the first round. At the moment, it's open-ended.

Reply:
Agreed

Reply:
https://daodao.zone/dao/juno1rw92sps9q4mm7ll3x9apnunlckchmn3v7cttchsf48dcdyajzj2sajfxcn/proposals/A3

Reply:
Epoch period has been updated.
https://github.com/CosmosContracts/delegations/pull/1/commits/a66ea4ebc133a41110469f9e140e5341e6b3cf80